components-comparison.html

```diff
@@ -0,0 +1,1484 @@
+                            </summary>
+                            <code>&lt;drawer&gt;</code>, <code>&lt;drawer-navigation&gt;</code>
+                        </details>
+                    </td>
+                    <td class="c4 no"></td>
+                    <td class="c5 no"></td>
+                </tr>
+                <tr>
+                    <td class="c0">Built-in <abbr title="Components">Comp.</abbr></td>
+                    <td class="c1">match-media</td>
+                    <td class="c2 no"></td>
+                    <td class="c3 no"></td>
+                    <td class="c4"><code>&lt;match-media&gt;</code></td>
+                    <td class="c5 no"></td>
+                </tr>
+                <tr>
+                    <td class="c0">Built-in <abbr title="Components">Comp.</abbr></td>
+                    <td class="c1">aria-component</td>
+                    <td class="c2 no"></td>
+                    <td class="c3 no"></td>
+                    <td class="c4"><code>&lt;aria-component&gt;</code></td>
+                    <td class="c5 no"></td>
+                </tr>
+                <tr>
+                    <td class="c0">Built-in <abbr title="Components">Comp.</abbr></td>
+                    <td class="c1">page metadata</td>
+                    <td class="c2 no"></td>
+                    <td class="c3 no"></td>
+                    <td class="c4"><code>&lt;page-meta&gt;</code></td>
+                    <td class="c5 no"></td>
+                </tr>
+                <!-- Data Binding -->
+                <tr class="large-cell">
+                    <td class="c0">Data binding</td>
+                    <td class="c1">Text interpolation</td>
+                    <td class="c2 no">&nbsp;</td>
+                    <td class="c3">
+                        <details>
+                            <summary>
+                                Vue.js-like approach under the same .ux document.
+                            </summary>
+                            <code>
+                                <pre>
+&lt;template&gt;
+  &lt;text&gt;{{message}}&lt;/text&gt;
+&lt;/template&gt;
+
+&lt;script&gt;
+  export default {
+    private: {
+      message: 'Hello'
+    }
+  }
+&lt;/script&gt;</pre>  
+                            </code>
+                            <a href="https://doc.quickapp.cn/framework/template.html">More information</a>.
+                        </details>
+                    </td>
+                    <td class="c4">
+                        <details>
+                            <summary>
+                                Dynamic data in AXML is bound to the Page corresponding data content.
+                            </summary>
+                            <code>
+                                <pre>
+&lt;view&gt; {{ message }} &lt;/view&gt;
+
+Page({ 
+   data: { 
+     message: 'Hello alipay!', 
+    }, 
+});
+                                </pre>  
+                            </code>
```

Comment:
```suggestion
                            <pre><code>&lt;view&gt; {{ message }} &lt;/view&gt;

Page({ 
   data: { 
     message: 'Hello alipay!', 
    }, 
});</code></pre>
```

Should be `<pre><code>` per https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-code-element

Reply:
Same below.